src/constants.ts

```diff
@@ -64,6 +64,18 @@ export const BRIDGED_USDC_SYMBOLS = [
   TOKEN_SYMBOLS_MAP["USDC.e"].symbol,
   TOKEN_SYMBOLS_MAP.USDbC.symbol,
   TOKEN_SYMBOLS_MAP.USDzC.symbol,
+  TOKEN_SYMBOLS_MAP.USDB.symbol,
```

Comment:
This is wrong. We do _not_ want to map USDB to USDC since there is a whole slew of issues which can arise due to the differences in token decimals.

Reply:
https://github.com/across-protocol/sdk/pull/969/commits/c8d66d33d4a161e537b80f5489963899f1c4dced